Cargo.toml

```diff
@@ -26,6 +26,7 @@ members = [
     "examples/sqlite/getting_started_step_2",
     "examples/sqlite/getting_started_step_3",
 ]
+resolver = "2"
```

Comment:
Clippy was complaining that resolver was version 1, which apparently doesn't work with 2021 edition, so it was warning to switch to 2.

Reply:
That's what is causing that last CI issues. Seems like we should rather pull in https://github.com/diesel-rs/diesel/commit/b816eda5325daf27162ad1614ddbfc2e609cdfce to fix that.